src/components/DappBrowser/Homepage.tsx

```diff
@@ -331,7 +335,7 @@ export default function Homepage() {
               </Text>
             </Inline>
             <Inline space={{ custom: CARD_PADDING }}>
-              {trendingDapps.map(site => (
+              {[trendingDapps[trendingDapps.length - 4], trendingDapps[trendingDapps.length - 2]].map(site => (
```

Comment:
🤔

Reply:
Should we edit the list instead and include only the ones whe want to show?